src/org/omegat/core/statistics/FindMatches.java

```diff
@@ -578,4 +578,11 @@ private void checkStopped(IStopped stop) throws StoppedException {
     @SuppressWarnings("serial")
     public static class StoppedException extends RuntimeException {
     }
+
+    /**
+     * for unit test.
+     */
+    List<NearString> searchForTest(String searchSentence) {
+        return search(searchSentence, false, () -> false, false);
+    }
```

Comment:
It is better to implement in the Test class instead of main class with Mockito library.